qiita_pet/handlers/study_handlers/artifact.py

```diff
@@ -107,6 +107,16 @@ def patch(self):
         self.write(response)
 
 
+class ArtifactGetSamples(BaseHandler):
+    @authenticated
+    def get(self):
+        aids = map(int, self.request.arguments.get('ids[]', []))
```

Comment:
is this equivalent to `aids = map(int, self.get_arguments('ids'))`? I'm just trying to understand the differences of using one or another if they're not the same.